frontend/src/features/performance/PerformanceInitial.tsx

```diff
@@ -0,0 +1,137 @@
+  const setStartMass = useInitialStore((state) => state.setStartMass)
+  const setFuelMass = useInitialStore((state) => state.setFuelMass)
+  const setFuelcons = useInitialStore((state) => state.setFuelcons)
+  const setVmax = useInitialStore((state) => state.setVmax)
+  const setWmax = useInitialStore((state) => state.setWmax)
+  const setArea = useInitialStore((state) => state.setArea)
+  const setAspectRatio = useInitialStore((state) => state.setAspectRatio)
+  const setCx0 = useInitialStore((state) => state.setCx0)
+  const setCzmax = useInitialStore((state) => state.setCzmax)
+
+  const [calculatePower] = usePower();
+  const [maxPower, setMaxPower] = useState(calculatePower(0))
+
+  useEffect(() => {
+    setMaxPower(() => calculatePower(flightAltitude))
+  }, [flightAltitude, calculatePower])
+
+  return (
+    <div className="flex w-full p-4">
+      <div className="flex flex-col w-80 mr-8 space-y-1">
+        <InputSlider
+          value={propnumber}
+          setter={setPropnumber}
+          step={1}
+          min={1}
+          max={2}
+          label="Number of Engines"
+          unit=""
+        />
+        <InputNumber
+          value={nominalPower}
+          setter={setNominalPower}
+          step={0.1}
+          label="Nominal Power"
+          unit="kW"
+        />
+        <InputNumber
+          value={startMass}
+          setter={setStartMass}
+          step={0.01}
+          label="Start Mass"
+          unit="kg"
+        />
+        <InputNumber
+          value={fuelMass}
+          setter={setFuelMass}
+          step={1}
+          label="Mass of Fuel"
+          unit="kg"
+        />
+        <InputNumber
+          value={fuelcons}
+          setter={setFuelcons}
+          step={0.001}
+          label="SFC"
+          unit="kg/kWh"
+        />
+        <InputNumber
+          value={vmax}
+          setter={setVmax}
+          step={0.01}
+          label="Max Velocity"
+          unit="m/s"
+        />
+        <InputNumber
+          value={area}
+          setter={setArea}
+          step={0.001}
+          label="Wing area"
+          unit="m^2"
+        />
+        <InputNumber
+          value={aspectRatio}
+          setter={setAspectRatio}
+          step={0.001}
+          label="Aspect Ratio"
+          unit="-"
+        />
+        <InputNumber
+          value={cx0}
+          setter={setCx0}
+          step={0.0001}
+          label="Minimal Drag Coefficient"
+          unit="-"
+        />
+        <InputNumber
+          value={czmax}
+          setter={setCzmax}
+          step={0.001}
+          label="Maximal Lift Coefficient"
+          unit="-"
+        />
+        <InputSlider
+          label="Flight altitude"
+          unit="km"
+          value={flightAltitude}
+          step={0.1}
+          min={0}
+          max={10}
+          setter={setFlightAltitude} 
+        />   
+      </div>
+      <div>
+        <PerformanceInitialEnduranceChart/>
```

Comment:
The animated charts that I've been working on are almost out. We can integrate them right away as they become available